server/src/test/java/org/jetbrains/bsp/bazel/server/bloop/BspModuleExporterTest.kt

```diff
@@ -78,8 +78,8 @@ class BspModuleExporterTest {
             baseDirectory.toScalaList().toOption(),
             emptyScalaList(),
             scalaListOf(baseDirectory.resolve("cp1")),
-            bloopRoot.resolve("z_73CA8EDE4AC5"),
```

Comment:
how could it break?

Reply:
toString for `Label` has changed and the hash in bloop is different now

Reply:
imo no - i can use `Naming.compilerOutputNameFor` there